vimmo/API/endpoints.py

```diff
@@ -99,6 +101,20 @@ def get(self):
         # Retrieve the database connection
         db = get_db()
         # Initialize a query object with the database connection
+        query = Query(db.conn)
+
+        if panel_id:
+            panel_data = query.get_panel_data(panel_id=args.get("Panel_ID"), matches=args.get("Similar_Matches"))
+            if "Message" in panel_data:
+                return panel_data
+            gene_query={record["HGNC_ID"] for record in panel_data["Associated Gene Records"]}
+            gene_query="|".join(gene_query)
+        elif r_code:
+            panel_data = query.get_panels_by_rcode(rcode=args.get("Rcode"), matches=args.get("Similar_Matches"))
+            if "Message" in panel_data:
+                return panel_data
+            gene_query={record["HGNC_ID"] for record in panel_data["Associated Gene Records"]}
+            gene_query="|".join(gene_query)
         query = PanelQuery(db.conn)
```

Comment:
Syntax err

Reply:
i think i changed this so this is handled in a different place. Not sure will test it